### PR TITLE
Merge 1-7-stable into master

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,18 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
...
+  [#8765](https://github.com/CocoaPods/CocoaPods/issues/8765)
+
+* Ensure all embedded pod targets are copied over to the host target.  
+  [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
+  [#8608](https://github.com/CocoaPods/CocoaPods/issues/8608)
+
+* Scope app spec dependent targets when no dedup'ing targets.
+  [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
+  [#8585](https://github.com/CocoaPods/CocoaPods/pull/8585)
+
```
Comment (amorde) [inline CHANGELOG.md]: any reason for putting this in between other entries?

**Comment by dnkoutso** [comment]
yes they will go for .rc2.

This is just catching up master with all 1.7.0 changes so we dont fall too far behind.

File: CHANGELOG.md (hunk 1 above)
Comment (dnkoutso) [reply CHANGELOG.md]: done

File: spec/unit/target/pod_target_spec.rb [hunk 2]
```diff
@@ -631,14 +642,8 @@ module Pod
           scoped_pod_target.first.dependent_targets.first.name.should == 'OrangeFramework-Pods'
           scoped_pod_target.first.test_dependent_targets_by_spec_name.count.should == 1
           scoped_pod_target.first.test_dependent_targets_by_spec_name['OrangeFramework'].first.name.should == 'matryoshka-Pods'
-        end
-
-        it 'scopes app dependencies' do
```
Comment (dnkoutso) [inline spec/unit/target/pod_target_spec.rb]: this was already fixed on master. I only took a tiny part of the 1-7-stable fix and merged the two. The same test is covered by the one above so its all good.

**Pull request merged**